backend/simpleLambda/index.js

```diff
@@ -1,51 +1,52 @@
-const messafe='null'
+exports.handler = async (event) => {
+  console.log("event: " + JSON.stringify(event));
+  console.log("----- lambda start");
```

Comment:
Do you need all these console logs still?  Typically you'd want to remove all console logs before pushing into a main branch. _(there are two more on lines 8 and 9)_

Reply:
No, I am gonna clean the code .